src/components/CurrencyInput.js

```diff
@@ -139,16 +141,27 @@ class CurrencyInput extends PureComponent<Props, State> {
       placeholder,
     } = this.props;
     const { displayValue } = this.state;
+
+    // calculating an approximative font size
+    const screenWidth = Dimensions.get("window").width * 0.75;
+    const dynamicFontSize = Math.round(
+      clamp(
+        Math.sqrt((screenWidth * 32) / displayValue.length),
+        8,
+        isActive ? 32 : 24,
+      ),
```

Comment:
that's a bit dangerous as i doubt we use a monospace font. mmmh

Reply:
Yes but sadly it works only for <Text /> :(

The font size is capped between 8 and 32 so it can't really get out of hand.

Reply:
Sadly the native way only works for text components and this is actually an input ...

So another solution would be to replace inputs by texts whenever the send max is on.

In this PR's approach, the font size is capped between 8 and 32 so it cannot really get out of hand.